cli-plugins/apps/command-reference/clustersupplychain.hbs.md

```diff
@@ -0,0 +1,43 @@
+# Tanzu Apps Cluster Supply Chain
+
+## tanzu apps cluster supply chain get
+
+`tanzu apps clustersupplychain get` command is used to get a detailed information of the cluster supply chain.
+
+### Default view
+
+The default view of `tanzu apps clustersupplychain get` command shows the status of the supply chain, and the selectors that a workload must match so it's taken by that supply chain.
+
```

Comment:
Add "See [clustersupplychain get](link to that command here) for a more detailed view of the selectors and conditions which must be met for a workload to be selected by a specific supply chain"